crates/swc_css_codegen/tests/fixture/values/ident/output.min.css

```diff
@@ -1 +1 @@
-:root{--fg-color:blue;--fg-\color-other:blue}.box{background-color:rebeccapurple;background-color:var(--fg-color);border-radius:10px;width:100px;height:100px}.box:hover{animation-name:rotate\ rotate;animation-duration:.7s}@keyframes rotate\ rotate{0%{transform:rotate(0)}100%{transform:rotate(360deg)}}@keyframes rotate{0%{transform:rotate(0)}100%{transform:rotate(360deg)}}@keyframes rotat\e{0%{transform:rotate(0)}100%{transform:rotate(360deg)}}*|*{color:blue}*|*{color:blue}@media screen and (min-width:0\0){body{color:red}}body{margin:-\1px}
+:root{--fg-color:blue;--fg-\color-other:blue}.box{background-color:rebeccapurple;background-color:var(--fg-color);border-radius:10px;width:100px;height:100px}.box:hover{animation-name:rotate\ rotate;animation-duration:.7s}@keyframes rotate\ rotate{0%{transform:rotate(0)}100%{transform:rotate(360deg)}}@keyframes rotate{0%{transform:rotate(0)}100%{transform:rotate(360deg)}}@keyframes rotat\e{0%{transform:rotate(0)}100%{transform:rotate(360deg)}}*|*{color:blue}\*|*{color:blue}@media screen and (min-width:0\0){body{color:red}}body{margin:-\1px}
```

Comment:
Original code:
```css
\*|*{color:blue}
```

According spec it is `\*` namespace (so we print it as ident). 

Chrome has bug here `\*|*{color:blue}` parsed as `*{color:blue}`, firefox doesn't affected this bug and just can't parse it.

And both wrong here, chrome should not parse this as any namespace, firefox should parse this with `\*` namespace :smile: I will report this in both browsers